pkg/jobs/jobspb/wrap.go

```diff
@@ -127,6 +128,8 @@ func (rs ReplicationStatus) String() string {
 		return "replication cutting over"
 	case ReplicationError:
 		return "replication error"
+	case CreatingInitialSplits:
+		return "creating initial splits"
 	default:
 		return fmt.Sprintf("unimplemented-%d", int(rs))
```

Comment:
I don't recall why we felt making this an Int was important rather than some const string literals.  A bit of a bummer that every time we add these we'll display this on old nodes.